mne/io/edf/edf.py

```diff
@@ -20,6 +20,49 @@
 from ..constants import FIFF
 from ...filter import resample
 from ...externals.six.moves import zip
+from ...utils import copy_function_doc_to_method_doc
+
+
+def get_edf_events(raw):
```

Comment:
We probably should document the classes, since some have methods that are specific to them (and this could grow).